### DISCUSSION TITLE
7flux1-schnell-Q5_0.gguf Model type: Unknown

### DISCUSSION THREAD

**Discussion by gerroon** [runew0lf/RuinedFooocus#189] [Q&A] [answered]
Hi

I downloaded couple of these models based on the recommendations from https://github.com/runew0lf/RuinedFooocus/wiki/Flux-%E2%80%90-GGUF-and-unet-safetensors

I have a 12gb 3080

```
Hashing c:\RuinedFooocus\modules\..\models\checkpoints\FLUX\7flux1-schnell-Q5_0.gguf                                                                             

Error: Model 7flux1-schnell-Q5_0.gguf Not Found on civit.ai  
```

But I am getting an error it says unknown type. Which base models am I supposed to download? The models are in the folder, and it clearly lists them in the UI

![image](https://github.com/user-attachments/assets/49b35c53-1d73-400d-976b-31e290f36976)

![image](https://github.com/user-attachments/assets/c760e04c-46a4-49f6-9c01-c9446f6f71bf)

**Comment by yownas** [comment, marked as answer]
The short answer is: Don't worry. Everything is ok.

The error is just that RF couldn't find the model on CivitAI and was unable to get a nice preview for it. So you'll get the "missing preview" image instead. You can replace that image if you want to, in `cache\checkpoints\` there should be an image with the same name as the model, just replace it.

The issue that the model is "Unknown"  is also harmless, in the future we might try to make things a bit smarter where LoRAs could be filtered to match the checkpoint-type and things like that, but before that we need to figure out a better way than to rely on CivitAI for that info.

So... Don't worry. Ignore the error. Replace the preview image if you want to.
